docs/webhooks/index.html

```diff
@@ -15,7 +16,8 @@
         <div class="header">
             <div class="header-left">
                 <a href="https://codesignal.com" class="logo-link">
-                    <img alt="CodeSignal home" src="https://codesignal.s3.amazonaws.com/uploads/1553120624174/logo_main.svg">
+                    <img alt="CodeSignal home" height="50px" src="https://theme.zdassets.com/theme_assets/9544451/012a31c7f1f2ddccf5a4424fb417c11f94315a77.png">
+                                                <!--   https://codesignal.s3.amazonaws.com/uploads/1553120624174/logo_main.svg -->
```

Comment:
Same comment about the commented-out URL here -- okay to just delete remorselessly. 😄 

For the new link, I'm not sure I like that we're hotlinking to something from `zdassets` (Zendesk assets?) instead of to a domain that we control directly or something like S3 that's meant for static hosting. I don't know that much about our Zendesk setup, so maybe it's okay, but here's an alternate logo image we could use that would make me feel safer:
https://app.codesignal.com/img/logos/logo_blue.svg